src/enrich/sled.rs

```diff
@@ -174,7 +174,7 @@ impl Worker {
         let utxo_refs: Vec<_> = txs
             .iter()
             .flat_map(|tx| tx.inputs())
-            .filter_map(|input| input.output_ref())
+            .map(|input| input.output_ref())
```

Comment:
is this some bug fix?

Reply:
It's to accommodate for a change upstream. The `output_ref` call now returns the value without a Result<> wrapper. This code has the same effect as before.